app/src/components/markers/tinyPinMarker.scss

```diff
@@ -3,16 +3,17 @@
 $markerSize: 12px;
 $markerSizehovered: 18px;
 .tiny-marker {
-    width: $markerSize;
-    height: $markerSize;
-    border-radius: 100%;
-    background: $lightBlue;
-    margin-top: -2px;
-    margin-left: -3px;
-    border: 2px solid $lighterGrey;
+  width: $markerSize;
```

Comment:
please set in your IDE tab as 4 spaces, not 2

Reply:
sure